### PR TITLE
feat(web): add git integration guide

### PR DESCRIPTION
**Pull request by antonymarion**
ref: MANAGER-14111

| Question         | Answer
| ---------------- | ---
| Branch?          | `master` /
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Feat#14111
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
~~- [ ] Breaking change is mentioned in relevant commits~~

### CODE REVIEW

File: packages/manager/apps/web/client/app/hosting/multisite/git-integration/association/git-association.html [hunk 1]
```diff
@@ -129,6 +140,31 @@ <h3 data-translate="hosting_multisite_git_association_configure_auto"></h3>
...
+    <a
+        class="oui-link_icon row m-0 p-0"
+        target="_blank"
+        rel="noopener"
+        data-ng-href="{{ ::$ctrl.guide.url }}"
+    >
+        <span
+            data-translate="hosting_multisite_git_association_guide_configure_webhook"
+        ></span>
+        <span class="oui-icon oui-icon-arrow-right" aria-hidden="true"></span>
```
Comment (darsene) [inline packages/manager/apps/web/client/app/hosting/multisite/git-integration/association/git-association.html:153]:
```suggestion
        <span class="oui-icon oui-icon-external-link" aria-hidden="true"></span>
```

File: packages/manager/apps/web/client/app/hosting/multisite/git-integration/association/git-association.html [hunk 2]
```diff
@@ -129,6 +140,31 @@ <h3 data-translate="hosting_multisite_git_association_configure_auto"></h3>
...
+    <a
+        class="oui-link_icon row m-0 p-0"
+        target="_blank"
+        rel="noopener"
+        data-ng-href="{{ ::$ctrl.guide.url }}"
+    >
+        <span
+            data-translate="hosting_multisite_git_association_guide_configure_tutorial"
+        ></span>
+        <span class="oui-icon oui-icon-arrow-right" aria-hidden="true"></span>
```
Comment (darsene) [inline packages/manager/apps/web/client/app/hosting/multisite/git-integration/association/git-association.html:164]:
```suggestion
        <span class="oui-icon oui-icon-external-link" aria-hidden="true"></span>
```

File: packages/manager/apps/web/client/app/hosting/multisite/git-integration/association/git-association.html [hunk 3]
```diff
@@ -129,6 +140,31 @@ <h3 data-translate="hosting_multisite_git_association_configure_auto"></h3>
     </div>
 </form>
 
+<div class="mb-5">
+    <a
+        class="oui-link_icon row m-0 p-0"
+        target="_blank"
+        rel="noopener"
+        data-ng-href="{{ ::$ctrl.guide.url }}"
```
Comment (ghyenne) [inline packages/manager/apps/web/client/app/hosting/multisite/git-integration/association/git-association.html:148]:
```suggestion
        data-ng-href="{{ ::$ctrl.GIT_ASSOCIATION_GUIDE_LINK }}"
```

File: packages/manager/apps/web/client/app/hosting/multisite/git-integration/association/git-association.html (hunk 1 above)
Comment (antonymarion) [reply packages/manager/apps/web/client/app/hosting/multisite/git-integration/association/git-association.html:153]: seems arrow is required here cf "maquette" 

image

**Review by darsene — APPROVED** [review]

**Pull request merged**